sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/HttpTimeoutPolicyControlPlaneHotPath.java

```diff
@@ -18,8 +18,8 @@ private HttpTimeoutPolicyControlPlaneHotPath() {
     }
 
     public List<ResponseTimeoutAndDelays> getTimeoutList() {
```

Comment:
Not introduced by this PR but perhaps the unused import can be removed.

Reply:
Done